games-action/polymc/polymc-1.4.1-r2.ebuild

```diff
@@ -81,6 +81,9 @@ DEPEND="
 # And we need more than just the GL headers
 RDEPEND="
 	${COMMON_DEPENDS}
+
+	>=dev-qt/qtsvg-${MIN_QT}:${QT_SLOT}
```

Comment:
When we add a new runtime dependency to an existing ebuild we need to revision bump the ebuild to ensure that the new runtime dependency is pulled in for all users.

Reply:
ook! I went ahead and removed 1.4.1 too, to keep things simpler.